src/write_influxdb_udp.c

```diff
@@ -326,22 +326,30 @@ static void flush_buffer(void) {
   write_influxdb_udp_init_buffer();
 }
 
+static void fill_send_buffer(char const *buffer, size_t len) {
```

Comment:
I think that `len <= wifxudp_config_packet_size` is ensured by the calling code, but it's not apparent when reading this function. I'd recommend adding an assertion to document this fact.

Reply:
Ok, done!